src/components/SearchBar/SearchBar.docs.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Primary, Controls, Story } from '@storybook/blocks';
+import * as SearchBarStories from './SearchBar.stories';
+
+<Meta of={SearchBarStories} />
+
+# SearchBar Component
+
+A reusable and customizable search bar component built with Material-UI. Features debounced input handling, keyboard navigation support, and accessible design patterns.
+
+## Overview
+
+The SearchBar provides a user-friendly search interface with:
+- **Debounced search queries** to optimize performance
+- **Material-UI integration** with consistent styling
+- **Keyboard support** (Enter key triggers search)
+- **Accessibility features** including ARIA labels
+- **Customizable styling** via theme props
+- **Comprehensive testing** with user interaction coverage
+
+## Basic Usage
+
+```tsx
+import { SearchBar } from './SearchBar';
+
+const MyComponent = () => {
+  const handleSearch = (query: string) => {
+    console.log('Search query:', query);
+  };
+
+  return (
+    <SearchBar 
+      onSearch={handleSearch}
+      placeholder="Search content..."
+    />
+  );
+};
```

Comment:
File ends with "No newline at end of file" in the patch. Add a trailing newline to comply with standard POSIX/Unix text file conventions and avoid noise in diffs.

**Sugerencia:**
```typescript
Add a newline at EOF.
```